src/RA_Dlg_Memory.cpp

```diff
@@ -838,7 +838,8 @@ INT_PTR Dlg_Memory::MemoryProc(HWND hDlg, UINT nMsg, WPARAM wParam, LPARAM lPara
             {
                 Col.iOrder = i;
                 Col.iSubItem = i;
-                Col.pszText = _T("Search Result");
+                ra::tstring str{ _T("Search Result") };
```

Comment:
```
Error C2440 '=': cannot convert from 'const char [14]' to 'LPSTR'			
         E0513 a value of type "const char *" cannot be assigned to an entity of type "LPSTR"
```
Basically the `rvalues `type does not match the `lvalues `type, this makes sure they do so the compiler knows we intend to drop the `const `qualifier.